### DISCUSSION TITLE
Invert Cover / Blind buttons and percentage

### DISCUSSION THREAD

**Discussion by thesebastianf** [zwave-js/zwave-js-ui#2475] [Ideas]
hey
second and third most wanted features in ZwaveJS from the HA comunity are about inverting buttons and percentage.
Is there a chance to get it pushed?

https://community.home-assistant.io/t/zwave-js-request-to-invert-covers/303156
https://community.home-assistant.io/t/configurable-cover-entities-to-invert-close-open/409595

to re-add the capability from the past like:
```
cover.fibaro_system_fgrm222_roller_shutter_controller_2_level:
invert_openclose_buttons: true
invert_percent: true

```
thank you

**Comment by robertsLando** [comment]
Are you using zwavejs integration? If so you should open your issue on hass repo not here, I have no control on how they create their entities

**Reply by thesebastianf** [reply]
Yes. OK thank you 👍🙏
